src/main.rs

```diff
@@ -114,10 +114,22 @@ fn execute(args: CliArguments) -> Result<FormatStatus> {
             #[cfg(feature = "completion")]
             cli::Command::Completions { shell } => {
                 use clap::CommandFactory;
+                use std::env::{args_os, current_exe};
+
+                let bin_path = args_os()
+                    .next()
+                    .map(PathBuf::from)
+                    .or_else(|| current_exe().ok());
```

Comment:
why should we use both args and current exe?

Reply:
I feel like using `"typstyle"` directly is much more easlier

Reply:
I see, symlink is beyond the scope of our consideration. The users can recompile or replace text with `sed` if they need to change this.